components/SearchBar.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { View } from 'react-native';
+import { Searchbar, List, Card } from 'react-native-paper';
+import { t } from 'react-native-tailwindcss';
+
+function SearchBar({ searcher, latitude, longitude, radius }) {
+  const [searchSuggestions, setSearchSuggestions] = useState([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const onChangeSearch = (query) => {
+    setSearchTerm(query);
+  };
+
+  const loadData = async () => {
+    const apiData = await fetch(
+      `http://localhost:8000/tags?latitude=${latitude}&longitude=${longitude}&radius=${radius}&searcher=${searcher}`
+    );
+    const responseText = await apiData.text();
+    const { tags } = JSON.parse(responseText);
+    setSearchSuggestions(tags || []);
+  };
+
+  useEffect(() => {
+    loadData();
+  }, []);
+
+  const matchTags = (tag) => {
+    const re = new RegExp(`${searchTerm}`, 'i');
+    return tag.match(re);
+  };
+
+  const handleItemPress = (text) => {
+    setSearchTerm(text);
+    console.log(text);
+  };
+
+  const handleSearchPress = () => {
+    console.log('searched');
+  };
+
+  return (
+    <>
+      <Searchbar
+        placeholder="Search"
+        onChangeText={onChangeSearch}
+        value={searchTerm}
```

Comment:
two or more search term would return no results due to each term separated by space (I assume)